app/controllers/owners_controller.rb

```diff
@@ -0,0 +1,36 @@
+class OwnersController < ApplicationController
```

Comment:
La convención en el orden de los controllers es: Index Show New Create Edit Update Delete (ISNCEUD)
Para la vida el acrónimo es:
I 
Should
Never
Call
Ex
Upon
Drinking